hs_core/hydro_realtime_signal_processor.py

```diff
@@ -0,0 +1,96 @@
+from django.db import models
+from haystack.signals import RealtimeSignalProcessor
+from haystack.exceptions import NotHandled
+import logging
+import types
+from haystack.query import SearchQuerySet
+from haystack.utils import get_identifier
+
+logger = logging.getLogger(__name__)
+
+
+class HydroRealtimeSignalProcessor(RealtimeSignalProcessor):
+
+    """ 
+    Customized for the fact that all indexed resources are subclasses of BaseResource. 
+
+    Notes: 
+    1. RealtimeSignalProcessor already plumbs in all class updates. We might want to be more specific. 
+    2. The class sent to this is a subclass of BaseResource, or another class. 
+    3. Thus, we want to capture cases in which it is an appropriate instance, and respond. 
+    """
+
+    def hydro_resource_is_present(self, instance): 
```

Comment:
Right.  My quick peek at the django-haystack code suggests that it catches any error with no more than a complaint in the logs.